msal4j-sdk/changelog.txt

```diff
@@ -1,3 +1,9 @@
+Version 1.13.8
+=============
+- Added support for CIAM authority
+- Better exception handling in interactive flow
+- Updated vulnerable dependency versions
```

Comment:
Also, added refresh_in logic for the managed identity flow. https://github.com/AzureAD/microsoft-authentication-library-for-java/commit/d345e619c30a3480f076d60a9d06af0fcf658e8f